README.md

```diff
@@ -132,7 +132,18 @@ public function main() {
             return;
         }
     }
-    
+
+    // Compress and upload file to a remote server
+    io:ReadableByteChannel|error inputChannel = io:openReadableFile("<Local data source path>");
+    if(inputChannel is io:ReadableByteChannel){
```

Comment:
Format code. Please apply for the rest of the file as well

Reply:
Fixed.